src/components/Search.component.jsx

```diff
@@ -0,0 +1,169 @@
+import React from "react";
+
+export const Search = (props) => {
+  //size = sm|md|lg
+  //variant = primary|secondary
+  const { size, variant } = props;
+  switch (variant) {
+    case "primary":
+      switch (size) {
+        case "sm":
+          return (
+            <label htmlFor="search" className="relative">
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                fill="none"
+                viewBox="0 0 24 24"
+                strokeWidth="1.5"
+                stroke="#EBEBEB"
+                class="pointer-events-none w-6 h-6 absolute top-1/2 transform -translate-y-1/2 right-3"
+              >
+                <path
+                  strokeLinecap="round"
+                  strokeLinejoin="round"
+                  d="M21 21l-5.197-5.197m0 0A7.5 7.5 0 105.196 5.196a7.5 7.5 0 0010.607 10.607z"
+                />
+              </svg>
+              <input
+                className="bg-primary-color hover:bg-primary-hover text-primary-buttonText m-1 p-2 text-sm rounded-xl placeholder-primary-buttonText focus:outline-none"
+                placeholder="Search"
```

Comment:
placeholder is already in props of HTML element so for greater component versatility you can delete 'placeholder'.

Reply:
ok will do :)